packages/documentation/src/stories/utilities/opacity/opacity.stories.ts

```diff
@@ -0,0 +1,38 @@
+import type { Args, StoryObj } from '@storybook/web-components';
+import { html } from 'lit';
+import './opacity.styles.scss';
+import { MetaExtended } from '@root/types';
+
+const meta: MetaExtended = {
+  id: '5e27e48d-a5f6-4e57-a343-7f40507fc27b',
+  title: 'Utilities/Opacity',
+};
+
+export const opacityOptions = [0, 25, 50, 75, 100];
+
+export default meta;
+
+type Story = StoryObj;
+
+export const Opacity: Story = {
+  args: {
+    opacity: '100',
+  },
+  argTypes: {
+    opacity: {
+      name: 'opacity',
+      description: 'Set the opacity of the element',
+      control: {
+        type: 'select',
+      },
+      options: opacityOptions,
+    },
+  },
+  render: (args: Args) => {
+    return html`
+      <div class="opacity-${args.opacity}">
+        <div class="opacity-content">Opacity ${args.opacity}%</div>
```

Comment:
I would prefer not to have this class shown in the snippet since it can be misleading:
```suggestion
        Opacity ${args.opacity}%
```